runtime/account_test.go

```diff
@@ -864,6 +881,9 @@ func getAccountKeyTestRuntimeInterface(storage *testAccountKeyStorage) *testRunt
 
 			return accountKey, nil
 		},
+		log: func(message string) {
+			storage.logs = append(storage.logs, message)
```

Comment:
This functionality was ported from the custom log-capturing runtime interface implementation below